building/windows.md

```diff
@@ -0,0 +1,173 @@
+# Windows
+
+It is possible to build Phoenix-rtos images for all targets and run `ia32-generic-qemu` on Windows using Cygwin or WSL.
+
+## Cygwin
+
+In the beginning, we need to download and set the QEMU environment.
+
+To do that we need to simply go to QEMU official website and select mirror which is targeted to our system.
+
+* [QEMU](https://www.qemu.org/download/#windows)
+
+![OS bit selection](_images/qemu-os-select.png)
+
+Next up we need to download Cygwin from the official website. It is a collection of Open Source tools that provide
+functionality similar to a Linux distribution on Windows.
+
+* [Cygwin](https://cygwin.com/install.html)
+
+<details>
+<summary>Cygwin installation/update process (click to expand)</summary>
+&nbsp;
+
+* The installation process is in this case also a package manager. Cygwin will ask of type of installation in our case
+it will be from internet
+
+![cygwin-download-source](_images/cygwin-download-source.png)
+
+* After that install process will ask about placement of root and local package directory
+
+![cygwin-root-dir](_images/cygwin-root-dir.png)
+![cygwin-local-directory](_images/cygwin-local-directory.png)
+
+* Cygwin have capability of using host internet connection, but it must be configured with our preferences (In most
+scenario it will be System Proxy Settings)
+
+![cygwin-internet-connection](_images/cygwin-internet-connection.png)
+
+* Selection of mirror from where packages are taken is tricky, sometimes it needs to be changed to acquire
+
+![cygwin-mirror-selection](_images/cygwin-mirror-selection.png)
+
+* After selecting mirror we can easily select what
+packages we want. But in larger groups, this can be annoying that's why we only select two elements (They will be
+installed with dependencies) which are mandatory to use package manager `apt-cyg` in Cygwin terminal.
+

```

Comment:
I would add a point about running the installation process
Something like:  Once these are chosen, proceed to run the installation process